testdata/exact-hashes.txt

```diff
@@ -1,5 +1,5 @@
 00000000fffffffff803f807f807f80ff90ff90fb90c9980ffffffff00000000  puffy_white.png
-00000fe07ff8fff81ff8399831983bcc35ac303c384c3ffc0ef08660c003ffff  00133601.jpg
+00000fe07ff8fff81ff8399831983bcc35ac303c384c3ffc8ef00660c003ffff  00133601.jpg
```

Comment:
Here is the hash value:
https://github.com/commonsmachinery/blockhash-python/blob/master/test/data/00133601_16_2.txt